installer/MSIX/PackagingLayout.xml

```diff
@@ -26,6 +26,13 @@
         <File DestinationPath="modules\System.ValueTuple.dll" SourcePath="..\..\x64\Release\modules\System.ValueTuple.dll"/>
         <File DestinationPath="modules\System.Numerics.Vectors.dll" SourcePath="..\..\x64\Release\modules\System.Numerics.Vectors.dll"/>
         <File DestinationPath="modules\Microsoft.Bcl.AsyncInterfaces.dll" SourcePath="..\..\x64\Release\modules\Microsoft.Bcl.AsyncInterfaces.dll"/>
+        <File DestinationPath="modules\powerpreview.dll" SourcePath="..\..\x64\Release\modules\powerpreview.dll"/>
+        <File DestinationPath="VFS\Windows\Microsoft.NET\assembly\GAC_64\PreviewHandlerCommon\v4.0_1.0.0.0__1c68b0a247329c04\PreviewHandlerCommon.dll" SourcePath="..\..\x64\Release\modules\PreviewHandlerCommon.dll"/>
```

Comment:
Why do we need to use GAC instead of just placing the libraries in the same dir as powerpreview.dll?

Reply:
Yes they can be placed in same directory as `powerpreview.dll`. Updated the `PackagingLayout.xml` to place the dll in `modules` directory. 

The reason why I was placing the dll's in virtualized Gac before because for the assembly  resolver to locate .Net assembly[ it will look in the GAC or at the application directory](https://docs.microsoft.com/en-us/dotnet/framework/interop/registering-assemblies-with-com) or the filepath mentioned in `CodeBase` registry value in class registration. Since I changed the assemblies not be strongly named which is a requirement to register in GAC We cannot use the GAC approach.
The downside with mentioning file Path in `CodeBase` is that I have to hardcode the assembly path which have package name as well(`file:///C:\Program Files\WindowsApps\Microsoft.PowerToys_0.15.2.0_x64__8wekyb3d8bbwe\modules\MarkdownPreviewHandler.dll`) Although with GAC approach as well we will need to mention the `PublicKeyToken` in the file path as above.